src/file/file.grpc.ts

```diff
@@ -80,7 +80,17 @@ export class FileMethods {
     const file: IFile = await FileService.getById(id);
     return new ResFile(file);
   }
-
+/**
+   * Retrieves  files by many ids.
+   * @param call
+   * @param idList
+   */
+ public static async GetFilesByIDs(call: any): Promise<{ files: ResFile[] }> {
+  const ids: string[] = call.request.ids;
```

Comment:
you need to think about a case where one id isn't valid (such as: 'balala') - we will return the other files or we are going to fail the entire request? (look at the file.repo)